src/styles/calendar.scss

```diff
@@ -27,8 +27,9 @@ $fd-calendar-item-spacing: 0.0625rem !default;
 $fd-calendar-button-horizontal-space: 0.25rem !default; // This value is used to create additional vertical space on the button which should be created by currently not implemented "Touchable Area" of the button.
 $fd-calendar-background: var(--sapList_Background) !default;
 $fd-calendar-item-font-size: var(--sapFontMediumSize) !default;
```

Comment:
The font size should be `--sapFontSize` per specs. The `var(--sapFontMediumSize)` is not defined:
<img width="416" alt="Screen Shot 2022-04-01 at 3 47 14 PM" src="https://user-images.githubusercontent.com/39598672/161331617-3a2d42ad-9930-4cac-bd66-67c396721e67.png">